draft-ietf-quic-http.md

```diff
@@ -379,10 +379,11 @@ An HTTP message (request or response) consists of:
 3. optionally, trailing headers, if present (see {{!RFC7230}}, Section 4.1.2),
    sent as a single HEADERS frame.
 
-A server MAY send one or more PUSH_PROMISE frames (see {{frame-push-promise}})
-before, after, or interleaved with the frames of a response message. These
-PUSH_PROMISE frames are not part of the response; see {{server-push}} for more
-details.
+A server MAY send one or more PUSH_PROMISE (see {{frame-push-promise}}) and/or
```

Comment:
```suggestion
A server MAY send one or more PUSH_PROMISE (see {{frame-push-promise}}) or
```
My technical writing trainer taught me that "and/or" means "or".  And that using one word in place of two was a win.